frontend/src/common/components/NavigateBackButton.tsx

```diff
@@ -3,7 +3,7 @@ import React from "react";
 import {Button, IconArrowLeft} from "hds-react";
 import {useNavigate} from "react-router-dom";
 
-export default function NavigateBackButton(): JSX.Element {
+export default function NavigateBackButton({disabled = false}: {disabled?: boolean}): JSX.Element {
```

Comment:
I was debating the usefulness of it with the back button myself, but decided to include it as it was included in the other ones too (to which I can see clear use cases). This will be used in the upcoming front end validation extensively. It could be added then too, sure - but I think this is (with the non-back-buttons) an essential feature which was an oversight to not include in the first place. A very understandable one for sure tho, as we didn't have any use for it yet.